concepticondata/conceptlists/Smeltzer-1995-300.tsv

```diff
@@ -0,0 +1,301 @@
+Smeltzer-1995-300-163	163	sept	1704	SEVEN	seven
+Smeltzer-1995-300-164	164	huit	1705	EIGHT	eight
+Smeltzer-1995-300-165	165	neuf	1483	NINE	nine
+Smeltzer-1995-300-166	166	dix	1515	TEN	ten
+Smeltzer-1995-300-167	167	cultiver	1541	CULTIVATE	to cultivate
+Smeltzer-1995-300-168	168	enterrer	1719	BURY	to bury
+Smeltzer-1995-300-169	169	fer	621	IRON	iron
+Smeltzer-1995-300-170	170	sang	946	BLOOD	blood
+Smeltzer-1995-300-171	171	lèvres	478	LIP	lip
+Smeltzer-1995-300-172	172	menton	1510	CHIN	chin
+Smeltzer-1995-300-173	173	barbe	678	BEARD	beard
+Smeltzer-1995-300-174	174	épaule	1482	SHOULDER	shoulder
+Smeltzer-1995-300-175	175	aisselle	1886	ARMPIT	armpit
+Smeltzer-1995-300-176	176	poitrine	1592	CHEST	chest
+Smeltzer-1995-300-177	177	côte	801	RIB	rib
+Smeltzer-1995-300-178	178	dos	1291	BACK	back
+Smeltzer-1995-300-179	179	hanche	1745	HIP	hip
+Smeltzer-1995-300-180	180	cuisse	800	THIGH	thigh
+Smeltzer-1995-300-181	181	coude	981	ELBOW	elbow
+Smeltzer-1995-300-182	182	doigt	1303	FINGER	finger
+Smeltzer-1995-300-183	183	corps	1480	BODY	body
+Smeltzer-1995-300-184	184	plaie	1109	WOUND	wound
+Smeltzer-1995-300-185	185	veine	1924	VEIN	vein
+Smeltzer-1995-300-186	186	larmes	1651	TEAR (OF EYE)	tear
+Smeltzer-1995-300-187	187	excrément	675	FAECES (EXCREMENT)	excrement
+Smeltzer-1995-300-188	188	huile	1484	OIL (ORGANIC SUBSTANCE)	oil
+Smeltzer-1995-300-189	189	mère	1216	MOTHER	mother
+Smeltzer-1995-300-190	190	frère aîné	1759	OLDER BROTHER	older brother
+Smeltzer-1995-300-191	191	petite soeur	1761	YOUNGER SISTER	younger sister
+Smeltzer-1995-300-192	192	ami	1325	FRIEND	friend
+Smeltzer-1995-300-193	193	oncle (côté matèrnelle.)	3220	MATERNAL UNCLE	uncle (mother's side)
+Smeltzer-1995-300-194	194	invité	1782	GUEST	guest
+Smeltzer-1995-300-195	195	chef	301	CHIEFTAIN	chieftain
+Smeltzer-1995-300-196	196	âme	811	SOUL	soul
+Smeltzer-1995-300-197	197	lion	1386	LION	lion
+Smeltzer-1995-300-198	198	rat	1490	RAT	rat
+Smeltzer-1995-300-199	199	chauve-souris	1793	BAT	bat
+Smeltzer-1995-300-200	200	cheval	615	HORSE	horse
+Smeltzer-1995-300-201	201	pintade	456	GUINEA FOWL	guinea fowl
+Smeltzer-1995-300-202	202	lézard	632	LIZARD	lizard
+Smeltzer-1995-300-203	203	crocodile	1857	CROCODILE	crocodile
+Smeltzer-1995-300-204	204	ver	1219	WORM	worm
+Smeltzer-1995-300-205	205	abeille	665	BEE	bee
+Smeltzer-1995-300-206	206	fourmis	587	ANT	ant
+Smeltzer-1995-300-207	207	araignée	843	SPIDER	spider
+Smeltzer-1995-300-208	208	scorpion	1538	SCORPION	scorpion
+Smeltzer-1995-300-209	209	bâton	1295	STICK	stick
+Smeltzer-1995-300-210	210	épine	124	THORN	thorn
+Smeltzer-1995-300-211	211	fleur	239	FLOWER	flower
+Smeltzer-1995-300-212	212	fruit	1507	FRUIT	fruit
+Smeltzer-1995-300-213	213	mil	931	MILLET	millet
+Smeltzer-1995-300-214	214	maïs	506	MAIZE	maize
+Smeltzer-1995-300-215	215	banane	868	BANANA	banana
+Smeltzer-1995-300-216	216	arachide	2364	PEANUT	peanut
+Smeltzer-1995-300-217	217	herbe	606	GRASS	grass
+Smeltzer-1995-300-218	218	“la brousse“	1683	BUSH	bush
```

Comment:
Is there a reason why this in quotation marks? If I do a google search I get pictures of grass landscapes (German: Steppe).

Reply:
I found it confusing too since it's the only concept enclosed in quotation marks in the source. I checked with Aymeric and he said it's quite comparable to English 'bush'. The Google images, however, seem to refer to a certain type of vegetation, as you said. I think I'll unmap it for now.

Reply:
I think "bush" is also ambiguous in English. It can mean the plant or wild grass land.

Reply:
You're right but I guess the current concepticon definition points to an individual plant.